### PR TITLE
[Dendrite] Polylith ingress and config fixes

### LINKED ISSUE #1600: [dendrite] Ingress for polylithEnabled
### Helm chart name

dendrite

### Describe the solution you'd like

It would be nice to setup multiple ingress for every dendrite service (with the correct prefix routing)

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by Jonnobrow**
**Description of the change**

Add an ingress configuration with correct routing for each polylith service.

**Benefits**

Reduces the toil in getting a polylith setup ready to go.

**Possible drawbacks**

Config fixes may be breaking, although unlikely to be honest.

**Applicable issues**

- closes #1600 

**Additional information**

**Checklist** 
- [x] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Chart `artifacthub.io/changes` changelog annotation has been updated in `Chart.yaml`. See [Artifact Hub documentation](https://artifacthub.io/docs/topics/annotations/helm/#supported-annotations) for more info.
- [x] Variables have been documented in the `values.yaml` file.

### CODE REVIEW

**Review by onedr0p — APPROVED** [review]

**Comment by genofire** [comment]
Wow, nice work @onedr0p - thank you for this ;)

Could i ask for setting the `.well-known/matrix/{server,client}`

File: charts/incubator/dendrite/values.yaml
Comment (genofire) [inline charts/incubator/dendrite/values.yaml:135]: that `ingress` is duplicated -> see line 39

**Review by genofire — COMMENTED** [review]
nice with examples for traefik and nginx

File: charts/incubator/dendrite/values.yaml
Comment (Jonnobrow) [reply charts/incubator/dendrite/values.yaml:135]: Will remove when I add the well-known paths. Currently dendrite only supports the server one, I have raised a PR (matrix-org/dendrite#2551) to add the client route, so I will hold off on opening another PR for the chart until that is merged and released.

**Pull request merged**
